### PR TITLE
#44 Dead code with checking error

### PR DESCRIPTION
**Pull request by GnusinPavel**
#44 remove redundant check

### CODE REVIEW

**Comment by 0crat** [comment]
@GnusinPavel this pull request is too small, just 5 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @g4s8[/z](https://www.0crat.com/u/g4s8) please review this and merge or reject

**Comment by GnusinPavel** [comment]
@g4s8 please take a look )

File: github/repo_delete.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/g4s8/gitstrap/context"
 	"log"
 )
-
+// @todo #44:30min Remove unused variable
```
Comment (g4s8) [inline github/repo_delete.go:8]: @GnusinPavel what's this `@todo` for? Looks like it's not related to origin task

Comment (GnusinPavel) [reply github/repo_delete.go:8]: there is a hint about unused variable, so I decided to put a puzzle

**Comment by GnusinPavel** [comment]
> @GnusinPavel please add ticket reference to PR description to link it together, and see my comment about todo

done

File: github/repo_delete.go (hunk 1 above)
Comment (GnusinPavel) [reply github/repo_delete.go:8]: Should I remove this puzzle?

Comment (g4s8) [reply github/repo_delete.go:8]: @GnusinPavel if it's just a single line to remove, then you can do it right in PR. But if requires some bigger refactoring, then you can submit a separate ticket

Comment (GnusinPavel) [reply github/repo_delete.go]: fixed, yes it's just a single line

**Comment by GnusinPavel** [comment]
Does PR ready to merge?

**Comment by GnusinPavel** [comment]
@g4s8 ping

**Comment by GnusinPavel** [comment]
@g4s8 please merge PR and close the ticket, if all done correctly, thx.

**Comment by 0crat** [comment]
Job `gh:g4s8/gitstrap#45` is not assigned, can't get performer
